### PR TITLE
[HACBS-221] Refactor the policies with violation format

### PR DESCRIPTION
**Pull request by dirgim**
* Switch deny rules to violations
* Add name, description and url to the policy details
* Refactor the existing messages

### CODE REVIEW

File: policies/image/policy/unsigned-rpms.rego [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
   unsigned_rpms := {rpm.nvra | rpm := input.rpms[_]; not rpm.gpg}
   not count(unsigned_rpms) == 0
 
-  msg = sprintf("All RPMs must be signed! Found following unsigned rpms(nvra): %s", [concat(", ", unsigned_rpms)])
-}
+  name := "image_unsigned_rpms"
+  msg = sprintf("All RPMs found on the image must be signed. Found following unsigned rpms(nvra): %s", [concat(", ", unsigned_rpms)])
+  description := "Providing packages signed with the secure Red Hat signing server indicates that the package was subject to all appropriate policies and procedures."
+  url := "https://docs.engineering.redhat.com/display/PRODSEC/PSP4.0+-+Offerings+Built+Securely"
+}
```
Comment (staticf0x) [inline policies/image/policy/unsigned-rpms.rego:11]:
```suggestion
}

```

**Review by staticf0x — APPROVED** [review]
LGTM

**Review by jsztuka — APPROVED** [review]
Lgtm

**Review by srivickynesh — APPROVED** [review]
LGTM

**Review by jsztuka — APPROVED** [review]
Great stuff, lgtm.

**Review by srivickynesh — APPROVED** [review]
LGTM

**Pull request merged**